DISCOVER/02_minimal_measures.md

```diff
@@ -2,7 +2,7 @@
 # Summary
 
 ## The easiest and most impactful measures
-Making your conference inclusive and encouraging diversity is a breeze! Just follow the measures below. The are explained in more depth in the other pages of this Cookbook. Measures that are **easy to implement** and have a **strong positive impact** are &quot;low-hanging fruit&quot; and marked with a red apple emoji: 🍎. Measures that take more effort but have considerable impact are marked with a green checkmark: ✅.
+Making your conference inclusive and encouraging diversity is achievable. Measures that are **easy to implement** and have a **strong positive impact** are &quot;low-hanging fruit&quot; and marked with a red apple emoji: 🍎. Measures that take more effort but have considerable impact are marked with a green checkmark: ✅.
```

Comment:
```suggestion
Making your conference inclusive and encouraging diversity is achievable. The measures below are explained in more depth in the other pages of this Cookbook. Measures that are **easy to implement** and have a **strong positive impact** are &quot;low-hanging fruit&quot; and marked with a red apple emoji: 🍎. Measures that take more effort but have considerable impact are marked with a green checkmark: ✅.
```

Reply:
Thanks! Can you add back in the second line, rewording similar to my suggestion above, correcting the typo, maybe combining the 2 sentences like:
>The measures below are explained in more depth in the other pages of this Cookbook.

Reply:
also you might have to resolve some conflicts, not sure if this edit makes sense anymore after #55  Sorry 😕